### ISSUE TITLE
Accessibility for Embed: Ensure frame titles are meaningful 

### ISSUE THREAD

**Issue by cjwest** [SU-SWS/decanter#409]
**Accessibility for Embed**
***Ensure frame titles are meaningful (Severity: 7)***

The YouTube iframe lacks an accessible name.

[User Impact]
Screen reader users will have no information about the content of the iframe. NOTE: the tester did hear JAWS seem to read the YouTube video title when announcing the iframe, but it is not visible in the code and this issue was flagged by the automated test engine. 

[Code Reference]
```
<iframe width="315" height="560" src="https://www.youtube.com/embed/_Nq4Z5i7lcs" frameborder="0" allowfullscreen=""/>
```
**To Reproduce**
1. http://decanter.stanford.edu/section-elements.html#kssref-elements-embedcontainer

**Expected behavior**
Provide a title attribute that is a short description of the contents of the frame.

[Code Example]
```
<iframe width="315" height="560" src="https://www.youtube.com/embed/_Nq4Z5i7lcs" frameborder="0" allowfullscreen="" title="Stanford University Campus Tour" />
```

**Additional context**
https://stanford.levelaccess.net/public/reporting/view_module.php?module_id=9634033

**Comment by sherakama** [comment]
Not a lot we can do about this here.
